test/functional/page_objects/settings_page.ts

```diff
@@ -177,6 +177,10 @@ export class SettingsPageObject extends FtrService {
     await this.testSubjects.click('deleteIndexPatternButton');
   }
 
+  async clickCloseFlyOutButton() {
```

Comment:
You ended up not using this so it should probably be deleted

Reply:
++ deleted